File: frontend/src/models/data/table.ts
export interface TableResponse {
  id: string;
  tableNumber: string;
  type: TableType;
  status: StatusTable;
  branchId: string;
}

export enum TableType {
  TABLE = 0,
  NINTENDO = 1,
}

export enum StatusTable {
  EMPTY = 0,
  NO_EMPTY = 1,
  COMING = 2,
}

export interface TableResponseHub {
  Id: string;
  TableNumber: string;
  Type: TableType;
  Status: StatusTable;
  BranchId: string;
}

File: Api/Boardgame/Boardgame.Domain/Entities/Table.cs
using Boardgame.Domain.Entities.Common;

namespace Boardgame.Domain.Entities;

public class TablePlay
{
    public Guid Id { get; set; } = Guid.NewGuid();
    public string TableNumber { get; set; } = string.Empty;
    public TableType Type { get; set; } = TableType.Table;
    public StatusTable Status { get; set; } = StatusTable.Empty;

    // Reference
    public Guid BranchId { get; set; }
    public Branch? Branch { get; set; }
    public ICollection<ScanSystem>? ScanSystems { get; set; }
    public Queue? Queue { get; set; }
}